CHANGELOG.md

```diff
@@ -1,4 +1,47 @@
-## 2.16.3 (Unreleased)
+## 2.17.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.2** (tested on **UserConnect-6.2.1700**)
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Implemented new resources to support CloudWAN:
+  - **aviatrix_device_aws_tgw_attachment**
+  - **aviatrix_device_interface_config**
+  - **aviatrix_device_registration**
+  - **aviatrix_device_tag**
+  - **aviatrix_device_transit_gateway_attachment**
+  - **aviatrix_device_virtual_wan_attachment**
+2. Implemented new resource to decouple ``domain_names`` out of ``aviatrix_fqdn``:
+  - **aviatrix_fqdn_tag_rule**        
+3. Implemented new resource to decouple ``policy`` out of ``aviatrix_firewall``:
+  - **aviatrix_firewall_policy**        
+4. Implemented new resources to support Multi-Cloud Segmentation:
+  - **aviatrix_segmentation_security_domain**        
+  - **aviatrix_segmentation_security_domain_connection_policy**        
+  - **aviatrix_segmentation_security_domain_association**        
+5. Implemented support of Editing for **aviatrix_saml_endpoint**
+6. Implemented support of ``subnet_size`` and ``num_of_subnet_pairs`` for advanced option for **aviatrix_vpc** and **aviatrix_vpc** data source
+7. Implemented support of ``enable_multicast`` as switch to enable/disable multi-cast for **aviatrix_aws_tgw**
+8. Implemented support of insane mode for GCP provider for **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+9. Implemented support of updating ``customized_routes`` and ``customized_route_advertisement`` for **aviatrix_aws_tgw**
+10. Implemented support of ``zone`` and ``ha_zone`` for Azure provider for **aviatrix_transit_gateway**
+11. Implemented support of ``zone`` and ``peering_ha_zone`` for Azure provider for **aviatrix_gateway**
+12. Implemented support of ``zone`` for Azure provider for **aviatrix_firewall_instance**
+13. Implemented new resource to decouple attaching **aviatrix_spoke_gateway** to **aviatrix_transit_gateway** out of **aviatrix_spoke_gateway**
+  - **aviatrix_spoke_transit_attachment**        
+14. Implemented support of ``enable_egress_transit_firenet`` for **aviatrix_transit_gateway**
+
+### Enhancements:
+1. Blocked updating ``allocate_new_eip``, ``eip`` and ``ha_eip`` for **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+2. Added ``private_subnets`` and ``public_subnets`` for **aviatrix_vpc** and **aviatrix_vpc** data source
```

Comment:
Added ``private_subnets`` and ``public_subnets`` as outputs for **aviatrix_vpc** resource and data source